File: javascript/6_kyu/how_many_pages.js
// https://www.codewars.com/kata/622de76d28bf330057cd6af8/train/python

function amountOfPages(summary) {
  let output = "";
  for (let i = 1; i < summary + 1; i++) {
    output += i.toString();
    if (output.length >= summary) return i;
  }
}

File: python/6_kyu/how_many_pages.py
# https://www.codewars.com/kata/622de76d28bf330057cd6af8/train/python

def amount_of_pages(summary):
    output = ""
    for i in range(1, summary + 1):
        output += str(i)
        if len(output) >= summary:
            return i


print(amount_of_pages(5))
